static/app/views/alerts/rules/metric/details/metricHistory.spec.tsx

```diff
@@ -13,10 +13,21 @@ describe('MetricHistory', () => {
   it('renders a critical incident', () => {
     render(<MetricHistory incidents={[TestStubs.Incident()]} />);
     expect(screen.getByRole('link', {name: '#123'})).toBeInTheDocument();
-    expect(screen.getByText('Number of Errors above 70')).toBeInTheDocument();
+    expect(screen.getByText('Number of errors above 70 in 1 hour')).toBeInTheDocument();
     expect(screen.getByText('12hr')).toBeInTheDocument();
   });
 
+  it('renders a critical % change incident', () => {
+    const incident = TestStubs.Incident();
+    incident.alertRule.comparisonDelta = 60;
+    render(<MetricHistory incidents={[incident]} />);
+    expect(
+      screen.getByText(
+        'Number of errors 70% higher in the last 1 hour compared to the same time one hour ago'
```

Comment:
I can't figure out what "same time one hour ago" is supposed to mean

Reply:
![image](https://github.com/getsentry/sentry/assets/1400464/825dff16-025b-4dfa-b34d-a07e0a3a64c7)